src/tools/pontoon/removing_users.md

```diff
@@ -0,0 +1,8 @@
+# Removing Pontoon users
+
+Access Django’s admin interface at `https://pontoon.mozilla.org/a/` (note that this is not the usual admin interface), then click `Users`. In the next page search for the user you want to deactivate (safer to search by email address):
+
+* Single user delete - Click on the user among the results displayed under the search field. Click `Delete` at the bottom of the page.
+* Bulk users delete - Select multiple users from users list page. Select the option of `Delete selected users` from actions dropdown and click `Go` button.
+
+Both of these above actions will show confirmation page. After click of `Yes`, users data will be removed from pontoon safely. We make sure to persist the data which we are interested in like Translations submitted by user, Comments given by the user etc.
```

Comment:
Note that you can also delete multiple users at once by selecting the checkboxes and choosing the `Delete selected users` action from the dropdown.